tests/Rector/Annotation/AnnotationTest.php

```diff
@@ -19,12 +19,6 @@ public function test(string $file): void
 
     public function provideDataForTest(): Iterator
     {
-        yield [__DIR__ . '/Fixture/inject.php.inc'];
-        yield [__DIR__ . '/Fixture/cascade.php.inc'];
-        yield [__DIR__ . '/Fixture/cascade_remove.php.inc'];
-        yield [__DIR__ . '/Fixture/ignorevalidation.php.inc'];
-        yield [__DIR__ . '/Fixture/lazy.php.inc'];
-        yield [__DIR__ . '/Fixture/transient.php.inc'];
-        yield [__DIR__ . '/Fixture/validate.php.inc'];
+        return $this->yieldFilesFromDirectory(__DIR__ . '/Fixture');
```

Comment:
This helper function makes sure no files are forgotten to run.

Also when a new fixture is added, people don't have to remember to put it here too.